Makefile

```diff
@@ -43,6 +43,8 @@ ifeq ($(LEDGER_ENABLED),true)
     endif
   endif
 endif
+build_tags += $(BUILD_TAGS)
+build_tags := $(strip $(build_tags))
 
```

Comment:
Why is this necessary?

Reply:
@bragaz Can we remove those lines?

Reply:
@bragaz Can we then add a comment to the Makefile just to remember why those are there in the future please?